src/components/CommentList.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import { getCommentsById } from "./utils/api";
+import CommentCard from "./CommentCard";
+
+const CommentList = props => {
+  const { article_id } = props;
+  const [comments, setComments] = useState([])
+  const [isLoading, setIsLoading] = useState(false)
+
+  useEffect(() => {
+    getCommentsById(article_id)
+      .then(({comments}) => {
+        setComments(comments)
+        setIsLoading(false)
+      })
+  })
+
+  return (
```

Comment:
What if there are no comments for an article. A message should be displayed, something like: 'No comments yet.'